### PR TITLE
backports: trivial 2024 01 01

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
## What was done?
trivial backports

## How Has This Been Tested?
Built and ran tests

## Checklist:
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation
- [x] I have assigned this pull request to a milestone _(for repository code-owners and collaborators only)_

### CODE REVIEW

File: test/README.md
Comment (knst) [inline test/README.md:125]: as I tested, it works.
But that's not true about `--jobs=100`:
dash's tests consumes much as 13Gb of storage for all tests, so, either 4Gb is not enough for 100 jobs nor 2.5Gb is enough for 32 jobs.

Let's bump it to 16 and 8?

Comment (knst) [inline test/README.md:125]:
```suggestion
For example running the test suite with `--jobs=100` might need a 16GB ramdisk, but running with `--jobs=4` will only need a 4GB ramdisk.
```

**Review by UdjinM6 — APPROVED** [review]
utACK (feel free to apply https://github.com/dashpay/dash/pull/5795#discussion_r1439414089 and merge without waiting for re-approval)

**Pull request merged**
